build.gradle

```diff
@@ -0,0 +1,50 @@
+group 'compiler-fleet'
+version '1.0-SNAPSHOT'
+
+buildscript {
+    ext.kotlin_version = '1.2.+'
+    ext.argparser_version = "2.+"
+
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'kotlin'
+apply plugin: 'application'
+
+repositories {
+    mavenCentral()
+    maven {
+        url "https://dl.bintray.com/xenomachina/maven"
+    }
+}
+
+dependencies {
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    compile "com.xenomachina:kotlin-argparser:$argparser_version"
+    compile 'com.google.cloud:google-cloud-pubsub:0.32.0-beta'
+}
+
+compileKotlin {
+    kotlinOptions.jvmTarget = "1.8"
+}
+compileTestKotlin {
+    kotlinOptions.jvmTarget = "1.8"
+}
+
+mainClassName = 'com.bardsoftware.backend.fleet.rmarkdown.SubscriberKt'
+
+jar {
```

Comment:
I think this one is not needed as well. Application plugin will do nice tarballs with everything which is required to run the app

Reply:
Done